src/main/java/parser/Parser.java

```diff
@@ -214,6 +219,30 @@ public String getDeadlineDateFromInput(String userInput) throws TantouException
         }
     }
 
+    //@@author sarahchow03
+    public int getQuantitySoldFromInput(String userInput) throws TantouException {
+        try {
+            command = ownParser.parse(options, getUserInputAsList(userInput));
+            return Integer.parseInt(command.getOptionValue(constants.Options.QUANTITY_OPTION));
+        } catch (ParseException e) {
+            throw new TantouException(String.format("Something went wrong when parsing: %s", e.getMessage()));
+        } catch (NumberFormatException e) {
+            throw new TantouException(String.format("Please enter a integer value for quantity sold!"));
```

Comment:
Perhaps we can let the user know exactly what format the program wants? The idea is that the user might not know what an `Integer` or `Double` is, so they might need to guess and check. Maybe something like `please enter a whole number` or `please enter a whole number or number with 2 decimal places` would be better? Up for discussion.